src/tracking.js

```diff
@@ -0,0 +1,8 @@
+var idSite = 8
+var piwikTrackingApiUrl = 'https://piwik.cozycloud.cc/piwik.php'
```

Comment:
Oh? No! That's a mistake.

My wish was to let the configuration under webpack's responsibility, but I could not configure webpack (such a noob I am!) to create the correct `tracking.js`.

As I said in [my previous comment](https://github.com/cozy/cozy-drive/pull/254#issuecomment-301110069), I hard-coded `siteId` and `URL` in `tracking.js`.